package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "thrift-pool",
+  "version": "1.0.0",
+  "description": "A module that wraps thrift interfaces in connection pooling logic to make them more resilient.",
+  "main": "index.js",
```

Comment:
Is there something inherently wrong with pre-publish steps?

The convention I've seen (and thought we followed) was to publish js so that people who want to use this library don't need to introduce a coffeescript dependency. That's what we do in understream, underscore.deep, unix-join, unix-sort, etc. It also doesn't necessarily involve a pre-publish step if you look at those repos.